src/makeRobotAccountant.js

```diff
@@ -7,6 +7,21 @@
 
 function makeRobotAccountant() {
   // write code here
+  let callCounter = 0;
+
+  const getSum = (num) => {
```

Comment:
you can return getSum function right away instead of declaring it first and then returning it ;)